lib/rendition.js

```diff
@@ -40,33 +41,39 @@ const Metadata = {
     TIFF_IMAGEHEIGHT: 'tiff:imageHeight'
 };
 
-// function fileHash(filename, algorithm) {
-//     return new Promise((resolve, reject) => {
-//         try {
-//             const shasum = crypto.createHash(algorithm);
-
-//             const stream = fs.createReadStream(filename);
-//             stream.on('data', data => shasum.update(data));
-//             stream.on('end', () => resolve(shasum.digest('hex')));
-//             stream.on('error', err => {
-//                 reject(`creating ${algorithm} hash failed: ${err.message || err}`);
-//             });
-//         } catch (error) {
-//             return reject(`creating ${algorithm} hash failed: ${error.message || error}`);
-//         }
-//     });
-// }
+function fileHash(filename, algorithm) {
```

Comment:
If following the model of newer classes using `Symbol` to mimic private/public methods, this could become a private method from the Rendition class.